tools/CMakeLists.txt

```diff
@@ -380,19 +404,59 @@ function(pico_embed_pt_in_binary TARGET PTFILE)
     )
 endfunction()
 
-# pico_encrypt_binary(TARGET AESFILE [SIGFILE])
+# pico_encrypt_binary(TARGET AESFILE IVFILE [SIGFILE <file>] [EMBED] [MBEDTLS] [OTP_KEY_PAGE <page>])
```

Comment:
Is it worth preemptively adding your `\param\` stuff from #2422 here too, or is it too early for that?

Reply:
I'll wait till that's merged, then rebase this and fix the issues that presumably pop up

Reply:
Merged and rebased